lib/user.rb

```diff
@@ -0,0 +1,41 @@
+class User < Sequel::Model(:userdetails)
+  User.unrestrict_primary_key
+  WORD_LIST = File.readlines(ENV['WORD_LIST_FILE']).map(&:strip)
+
+  def generate(email:)
+    existing_user = User.find(email: email)
+    return login_details(existing_user) if existing_user
+
+    user = create_user(email)
+    login_details(user)
+  end
+
+private
+
+  def random_username
+    username = ('a'..'z').to_a.sample(6).join
```

Comment:
Wonder if we can remove this duplication on line 19.

Reply:
I could do

```ruby
def generate_username
  ('a'..'z').to_a.sample(6).join
end
```
but I don't think that saves us much

Reply:
I think it looks a little nicer.